wso2-smoketest/.default.env

```diff
@@ -20,10 +20,20 @@ PROD_CLIENT_ID=your_prod_client_id
 PROD_CLIENT_SECRET=your_prod_client_secret
 
 APIS=(
-    # TODO: setup suite of default endpoints
-    "/internal/service1"
-    "/internal/service2"
-    "/external/service1"
-    "/external/service2"
-)o
-
+    # SIMULATOR
+    "/simulatorserver/2.0/simulations/response?status=200%20OK"
```

Comment:
Use default version instead? (in order to minimize maintenance?

Reply:
As the jira was written I was under the impression wanting version called was desired. And the script parses version from the url. 

But I agree using default versioning would make maintenance much easier.

Reply:
No, the API version is not important. 
If you men the output (where the version is printed) it just acts as an example.

Reply:
Then I'll remove the versions from the APIs and remove that part from the output.